carvis-web/src/server/routes/redisRoutes.js

```diff
@@ -0,0 +1,95 @@
+/* ======== REDIS SCHEMA =====
+places:
+---------
+hash name: place (string)
+keys:
+---
+userId:lat : int
+userId:lng : int
+userId:routable : string
+
```

Comment:
rather than caching by userId, which wont get much reusability, why not instead cache by city? so key is 'san francisco', value is an object whose keys are queries like 'mcdonalds' that were initiated by a user from san francisco. the value for 'mcdonalds' would be a list of mcdonalds locations that google returned for such a query. then once retrieved, we'd sort those mcdonalds locations by distance to that user.